src/App.js

```diff
@@ -4,7 +4,13 @@ import './App.scss';
 class App extends Component {
   render() {
     return (
-      <h1>Hello World!</h1>
+      <header className="mdc-top-app-bar">
```

Comment:
Since we're in JS land, lets use single quotes. We should add this to the eslinter.

Reply:
Done. This repo doesn't have an eslinter yet, I'll add that in another PR.